packages/plugins/plugin-inbox/src/components/Mailbox/MailboxContainer.tsx

```diff
@@ -14,15 +14,15 @@ import { StackItem } from '@dxos/react-ui-stack';
 import { TagPicker } from '@dxos/react-ui-tag-picker';
 
 import { InboxCapabilities } from '../../capabilities/capabilities';
-import { InboxAction, type MailboxType } from '../../types';
+import { InboxAction, type Mailbox } from '../../types';
 
 import { EmptyMailboxContent } from './EmptyMailboxContent';
-import { Mailbox, type MailboxActionHandler } from './Mailbox';
+import { type MailboxActionHandler, Mailbox as MailboxComponent } from './Mailbox';
```

Comment:
@thure this is the standard i'd like to establish -- that the types have the "natural" name, and that clashing components adjust (adding a "Component" suffix).